### ISSUE TITLE
Lazy loaded stack views are missing position information for screen readers

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3568]
## Describe the bug

Lazy loading stack views should provide position in set information for each stack item in view at any given time. 

## How to reproduce

https://clarity.design/documentation/stack-view

- Using a screen reader find the 'Lazy Loading of Children' example and read one of the 'stack-block-label' buttons. Observe that you don't hear any information regarding the data presented in 'stack-block-label' 

## Expected behavior

Use aria-level, aria-setsize, and aria-posinset as on each 'stack-block-label' button. 

For information on using aria-level, aria-setsize, and aria-posinset see https://www.w3.org/TR/wai-aria-practices-1.1/#TreeView

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by andreiconstantinbytex** [comment]
I have investigated the issue and there are two solutions: should I add the aforementioned aria-* attributes only on the documentation examples or directly on the 'clr-stack-block' component?

**Comment by gnomeontherun** [comment]
@andreiconstantinbytex I haven't dug into this, but if the aria attributes go on an internal DOM element inside of the clarity stack components, then we'll have to internalize it. If it can go directly onto the host, we can add this into the documentation with explanation for how apps should implement it themselves. 

It might be most helpful to do a little prototype that shows the desired behavior working in an isolated StackBlitz demo, then we can decide on the best approach to implement the functionality.
